test-lib-remote-openshift.sh

```diff
@@ -3,6 +3,11 @@
 # shellcheck source=/dev/null
 source "$(dirname "${BASH_SOURCE[0]}")"/test-lib.sh
 
+# this should be returned when something related to the openshift cluster
+# goes wrong during the test pipeline
+# shellcheck disable=SC2034
+readonly OC_ERR=11
```

Comment:
I agree with setting value to a different variable, but I am missing content, where it is used?

Reply:
In the PR comment is link to the nginx container, showing, how this could be used.